kb/skills/init/templates/scripts/render_okf_viewer.py

```
@@ -0,0 +1,297 @@
+        targets = set(c["related"]) | set(c["wikilinks"])
+        for t in targets:
+            if t in slugs and t != c["slug"]:
+                edge_set.add((c["slug"], t))
+        for href in c["md_links"]:
+            resolved = resolve_md_link(root, c["path"], href)
+            target_slug = path_to_slug.get(resolved) or (
+                os.path.basename(resolved)[:-3] if resolved.endswith(".md") else None
+            )
+            if target_slug and target_slug in slugs and target_slug != c["slug"]:
+                edge_set.add((c["slug"], target_slug))
+    degree = {s: 0 for s in slugs}
```

Ruff (0.15.20)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)